hazelcast/src/main/java/com/hazelcast/internal/memory/impl/DirectMemoryBits.java

```diff
@@ -16,16 +16,20 @@
 
 package com.hazelcast.internal.memory.impl;
 
-import sun.misc.Unsafe;
+import com.hazelcast.internal.memory.MemoryAccessor;
+
+import java.io.DataInput;
+import java.io.IOException;
+import java.io.UTFDataFormatException;
 
 /**
  * Utility class to read/write bits to given location (by base object/offset or native memory address)
  * by specified byte order (little/big endian).
  */
 public final class DirectMemoryBits {
+    private static final MemoryAccessor BYTE_BUFFER_MEMORY_ACCESSOR = MemoryAccessor.HEAP_BYTE_BUFFER_MEM;
+    private static final MemoryAccessor UNSAFE_MEMORY_ACCESSOR = MemoryAccessor.AMEM;
```

Comment:
We are trying to clean all `Unsafe` terms (names, classes, etc ...) in our code base and it is still in progress. So please don't use `Unsafe`. You may use `ALIGNED_MEMORY_ACCESSOR` here instead of `UNSAFE_MEMORY_ACCESSOR`